README.md

```diff
@@ -12,14 +12,66 @@ Storj is in the midst of a rearchitecture. Please stay tuned for our v3 whitepap
 
 Storj is a platform, token, and suite of decentralized applications that allows you to store data in a secure and decentralized manner. Your files are encrypted, shredded into little pieces and stored in a global decentralized network of computers. Luckily, we also support allowing you (and only you) to recover them!
 
-## To start developing
+# Start Using Storj
+
+### Download the latest release
+
+Go here to download the latest build
+
+### Configure AWS CLI
+
+In a new terminal session:
+
+Download and install the AWS CLI: https://docs.aws.amazon.com/cli/latest/userguide/installing.html
+
+Configure the AWS CLI:
+```bash
+$ aws configure
+AWS Access Key ID [None]: insecure-dev-access-key
+AWS Secret Access Key [None]: insecure-dev-secret-key
+Default region name [None]: us-east-1
+Default output format [None]: 
+$ aws configure set default.s3.multipart_threshold 1TB  # until we support multipart
+```
+Test some commands:
+
+### Upload an Object
+
+```bash
+$ aws s3 --endpoint=http://localhost:7777/ cp ~/Desktop/your-large-file.mp4 s3://bucket/your-large-file.mp4
+```
+
+### Download an Object
+
+```bash
+$ aws s3 --endpoint=http://localhost:7777/ cp s3://bucket/your-large-file.mp4 ~/Desktop/your-large-file.mp4
+```
+
+### Listing objects
+
+```bash
+aws s3 --endpoint=http://localhost:7777/ ls s3://bucket/ --recursive
+```
+
+### Generating a URL for an Object
```

Comment:
"Generate a URL for an Object"

Reply:
may be we should add delete as well, since it is commonly used - and we should be goood for common commands.